src/components/Categories/Categories.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
+import '../Style/Style.css';
```

Comment:
please make sure to write the import link exactly as the file route is. currently, you wrote `Style` as the CSS folder but you instead have a `style` folder. This create an error when running your app please fix it
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/37901469/232248813-0995a6ad-6958-4cfa-85a9-06752fe036ca.png">